bundles/org.openhab.core.model.script/src/org/openhab/core/model/script/actions/Voice.java

```diff
@@ -267,6 +270,14 @@ public static void startDialog(@ParamDoc(name = "keyword spotting service") @Nul
                 return;
             }
         }
+        org.openhab.core.voice.Voice prefVoice = null;
```

Comment:
Why don't you import that class?

Reply:
Because there is a conflict with another class Voice in the same package.